client/landing/stepper/declarative-flow/internals/steps-repository/__user/style.scss

```diff
@@ -2,11 +2,6 @@
 @import "@wordpress/base-styles/breakpoints";
 @import "@wordpress/base-styles/mixins";
 
-body.is-section-stepper .step-route {
-	//--color-accent: #117ac9;
-	--color-accent-60: #0e64a5;
-}
-
```

Comment:
@dc22 re https://github.com/Automattic/wp-calypso/pull/91182
@alshakero re https://github.com/Automattic/wp-calypso/pull/97371

:wave: Would one of you be able to provide the background on why this was needed in the first place and why the accent-60 was left behind in the most recent change?

I'm removing this rule to avoid a color mismatch on hover here:

https://github.com/user-attachments/assets/0c653d3e-7168-4583-9ee7-d029eee528c2

It doesn't seem to cause any regressions I can find but I could be wrong (I haven't looked around much more than the user step)

There does seem to be a button color mismatch between the login and the create account via email pages:

![Screenshot 2024-12-16 at 13-21-18 Create a site — WordPress com](https://github.com/user-attachments/assets/2c137a76-f7ef-4573-b7e8-396890ca7ee4)

![Screenshot 2024-12-16 at 13-21-25 Log In — WordPress com](https://github.com/user-attachments/assets/5a153d50-1f14-416c-8d5e-d2076c4a61ef)

But this seems unrelated to this change, is there something else leveraging this stepper-wide accent color?

Reply:
Thanks, I'll leave this removal in for now as I don't see any regressions with the user step.